go/worker/compute/executor/init.go

```diff
@@ -1,16 +1,52 @@
 package executor
 
 import (
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
 	workerCommon "github.com/oasisprotocol/oasis-core/go/worker/common"
 	"github.com/oasisprotocol/oasis-core/go/worker/compute"
 	"github.com/oasisprotocol/oasis-core/go/worker/registration"
 )
 
+const (
+	// CfgScheduleCheckTxEnabled enables checking each transaction before
+	// scheduling it.
+	CfgScheduleCheckTxEnabled = "worker.executor.schedule_check_tx.enabled"
+
+	cfgMaxQueueSize = "worker.executor.schedule_max_queue_size"
+)
+
+// Flags has the configuration flags.
+var Flags = flag.NewFlagSet("", flag.ContinueOnError)
+
+func scheduleCheckTxEnabled() bool {
```

Comment:
Since this is only used below, you could just use viper there directly instead of having these one-line functions?